syft/pkg/conan_lock_metadata_test.go

```diff
@@ -0,0 +1 @@
+package pkg
```

Comment:
Is this necessary?

Reply:
It would be if the test had made it in there! Let me see where that got stashed apologies

Reply:
Test added back Thanks for the review!